hedyorg/hedy
# [UI]Change the name of "program_header" for profile page

**Description**
**Changes** "program_header" in html from "my programs" to "programs".


Fixes (https://github.com/hedyorg/hedy/issues/4377)

**How to test**:

- Go to http://127.0.0.1:8080/my-profile
- In public profile, selecte the box that makes profile visible,  and update public profile.
- Go to https://127.0.0.1:8080/user/{your user name}
- See that program_header saying "Programs" instead of "My programs".

**Checklist**
